Tests don't run

~~~
stafford@stafford-pc MINGW64 /c/git/db-migrate-sqlite (allow_functions)
$ npm i
npm WARN deprecated minimatch@1.0.0: Please update to minimatch 3.0.2 or higher to avoid a RegExp DoS issue

> sqlite3@3.1.8 install C:\git\db-migrate-sqlite\node_modules\sqlite3
> node-pre-gyp install --fallback-to-build

[sqlite3] Success: "C:\git\db-migrate-sqlite\node_modules\sqlite3\lib\binding\node-v47-win32-x64\node_sqlite3.node" is installed via remote
db-migrate-sqlite3@0.2.1 C:\git\db-migrate-sqlite
+-- bluebird@3.4.6
+-- db-meta@0.4.1
+-- db-migrate-base@1.2.7
+-- db-migrate-shared@1.1.2
+-- sqlite3@3.1.8
| +-- nan@2.4.0
| `-- node-pre-gyp@0.6.31
|   +-- mkdirp@0.5.1
|   | `-- minimist@0.0.8
|   +-- nopt@3.0.6
|   | `-- abbrev@1.0.9
|   +-- npmlog@4.0.0
|   | +-- are-we-there-yet@1.1.2
|   | | +-- delegates@1.0.0
|   | | `-- readable-stream@2.1.5
|   | |   +-- buffer-shims@1.0.0
|   | |   +-- core-util-is@1.0.2
|   | |   +-- inherits@2.0.3
|   | |   +-- isarray@1.0.0
|   | |   +-- process-nextick-args@1.0.7
|   | |   +-- string_decoder@0.10.31
|   | |   `-- util-deprecate@1.0.2
|   | +-- console-control-strings@1.1.0
|   | +-- gauge@2.6.0
|   | | +-- aproba@1.0.4
|   | | +-- has-color@0.1.7
|   | | +-- has-unicode@2.0.1
|   | | +-- object-assign@4.1.0
|   | | +-- signal-exit@3.0.1
|   | | +-- string-width@1.0.2
|   | | | +-- code-point-at@1.0.1
|   | | | | `-- number-is-nan@1.0.1
|   | | | `-- is-fullwidth-code-point@1.0.0
|   | | |   `-- number-is-nan@1.0.1
|   | | +-- strip-ansi@3.0.1
|   | | | `-- ansi-regex@2.0.0
|   | | `-- wide-align@1.1.0
|   | `-- set-blocking@2.0.0
|   +-- rc@1.1.6
|   | +-- deep-extend@0.4.1
|   | +-- ini@1.3.4
|   | +-- minimist@1.2.0
|   | `-- strip-json-comments@1.0.4
|   +-- request@2.76.0
|   | +-- aws-sign2@0.6.0
|   | +-- aws4@1.5.0
|   | +-- caseless@0.11.0
|   | +-- combined-stream@1.0.5
|   | | `-- delayed-stream@1.0.0
|   | +-- extend@3.0.0
|   | +-- forever-agent@0.6.1
|   | +-- form-data@2.1.1
|   | | `-- asynckit@0.4.0
|   | +-- har-validator@2.0.6
|   | | +-- chalk@1.1.3
|   | | | +-- ansi-styles@2.2.1
|   | | | +-- escape-string-regexp@1.0.5
|   | | | +-- has-ansi@2.0.0
|   | | | | `-- ansi-regex@2.0.0
|   | | | +-- strip-ansi@3.0.1
|   | | | | `-- ansi-regex@2.0.0
|   | | | `-- supports-color@2.0.0
|   | | +-- commander@2.9.0
|   | | | `-- graceful-readlink@1.0.1
|   | | +-- is-my-json-valid@2.15.0
|   | | | +-- generate-function@2.0.0
|   | | | +-- generate-object-property@1.2.0
|   | | | | `-- is-property@1.0.2
|   | | | +-- jsonpointer@4.0.0
|   | | | `-- xtend@4.0.1
|   | | `-- pinkie-promise@2.0.1
|   | |   `-- pinkie@2.0.4
|   | +-- hawk@3.1.3
|   | | +-- boom@2.10.1
|   | | +-- cryptiles@2.0.5
|   | | +-- hoek@2.16.3
|   | | `-- sntp@1.0.9
|   | +-- http-signature@1.1.1
|   | | +-- assert-plus@0.2.0
|   | | +-- jsprim@1.3.1
|   | | | +-- extsprintf@1.0.2
|   | | | +-- json-schema@0.2.3
|   | | | `-- verror@1.3.6
|   | | `-- sshpk@1.10.1
|   | |   +-- asn1@0.2.3
|   | |   +-- assert-plus@1.0.0
|   | |   +-- bcrypt-pbkdf@1.0.0
|   | |   +-- dashdash@1.14.0
|   | |   +-- ecc-jsbn@0.1.1
|   | |   +-- getpass@0.1.6
|   | |   +-- jodid25519@1.0.2
|   | |   +-- jsbn@0.1.0
|   | |   `-- tweetnacl@0.14.3
|   | +-- is-typedarray@1.0.0
|   | +-- isstream@0.1.2
|   | +-- json-stringify-safe@5.0.1
|   | +-- mime-types@2.1.12
|   | | `-- mime-db@1.24.0
|   | +-- node-uuid@1.4.7
|   | +-- oauth-sign@0.8.2
|   | +-- qs@6.3.0
|   | +-- stringstream@0.0.5
|   | +-- tough-cookie@2.3.2
|   | | `-- punycode@1.4.1
|   | `-- tunnel-agent@0.4.3
|   +-- rimraf@2.5.4
|   | `-- glob@7.1.1
|   |   +-- fs.realpath@1.0.0
|   |   +-- inflight@1.0.6
|   |   | `-- wrappy@1.0.2
|   |   +-- inherits@2.0.3
|   |   +-- minimatch@3.0.3
|   |   | `-- brace-expansion@1.1.6
|   |   |   +-- balanced-match@0.4.2
|   |   |   `-- concat-map@0.0.1
|   |   +-- once@1.4.0
|   |   | `-- wrappy@1.0.2
|   |   `-- path-is-absolute@1.0.1
|   +-- semver@5.3.0
|   +-- tar@2.2.1
|   | +-- block-stream@0.0.9
|   | +-- fstream@1.0.10
|   | | `-- graceful-fs@4.1.9
|   | `-- inherits@2.0.3
|   `-- tar-pack@3.3.0
|     +-- debug@2.2.0
|     | `-- ms@0.7.1
|     +-- fstream@1.0.10
|     | +-- graceful-fs@4.1.9
|     | `-- inherits@2.0.3
|     +-- fstream-ignore@1.0.5
|     | +-- inherits@2.0.3
|     | `-- minimatch@3.0.3
|     |   `-- brace-expansion@1.1.6
|     |     +-- balanced-match@0.4.2
|     |     `-- concat-map@0.0.1
|     +-- once@1.3.3
|     | `-- wrappy@1.0.2
|     +-- readable-stream@2.1.5
|     | +-- buffer-shims@1.0.0
|     | +-- core-util-is@1.0.2
|     | +-- inherits@2.0.3
|     | +-- isarray@1.0.0
|     | +-- process-nextick-args@1.0.7
|     | +-- string_decoder@0.10.31
|     | `-- util-deprecate@1.0.2
|     `-- uid-number@0.0.6
`-- vows@0.8.0
  +-- diff@1.0.8
  +-- eyes@0.1.8
  `-- glob@4.0.6
    +-- graceful-fs@3.0.11
    | `-- natives@1.1.0
    +-- inherits@2.0.3
    +-- minimatch@1.0.0
    | +-- lru-cache@2.7.3
    | `-- sigmund@1.0.1
    `-- once@1.4.0
      `-- wrappy@1.0.2


stafford@stafford-pc MINGW64 /c/git/db-migrate-sqlite (allow_functions)
$ npm test

> db-migrate-sqlite3@0.2.1 test C:\git\db-migrate-sqlite
> node node_modules/.bin/vows

C:\git\db-migrate-sqlite\node_modules\.bin\vows:2
basedir=$(dirname "$(echo "$0" | sed -e 's,\\,/,g')")
          ^^^^^^^

SyntaxError: missing ) after argument list
    at exports.runInThisContext (vm.js:53:16)
    at Module._compile (module.js:387:25)
    at Object.Module._extensions..js (module.js:422:10)
    at Module.load (module.js:357:32)
    at Function.Module._load (module.js:314:12)
    at Function.Module.runMain (module.js:447:10)
    at startup (node.js:142:18)
    at node.js:939:3
npm ERR! Test failed.  See above for more details.
~~~

--- a/package.json
+++ b/package.json
@@ -4,7 +4,7 @@
   "description": "A sqlite driver for db-migrate",
   "main": "index.js",
   "scripts": {
-    "test": "node node_modules/.bin/vows"
+    "test": "vows"
   },
   "repository": {
     "type": "git",

